qucs/qucs/dialogs/simmessage.cpp

```diff
@@ -594,9 +594,6 @@ void SimMessage::startSimulator()
   QString sep(":");
 #endif
 
-  // append process PATH
-  // insert Qucs bin dir, so ASCO can find qucsator
-  env.insert("PATH", env.value("PATH") + sep + QucsSettings.BinDir );
```

Comment:
oh, I thought I added a comment but cannot see it anymore.

Now `asco` cannot find `qucsator` anymore. I have several versions of Qucs that I use in parallel, none is in the path. How can we make sure `asco` finds and run the right `qucsator`?